mkt/developers/views.py

```diff
@@ -1153,3 +1174,101 @@ def verify_data(self, data):
             return False
 
         return True
+
+
+class ContentRatingsPingbackV2(CORSMixin, SlugOrIdMixin, CreateAPIView):
+    """TODO: match v2 spec. This is currently just a stub."""
+    cors_allowed_methods = ['post']
+    parser_classes = (lib.iarc.utils.IARC_JSON_Parser,)
+    permission_classes = (AllowAny,)
+
+    queryset = Webapp.objects.all()
+    slug_field = 'app_slug'
+
+    def post(self, request, pk, *args, **kwargs):
+        log.info(u'Received IARC pingback for app:%s' % pk)
+
+        if request.content_type != 'application/json':
+            log.info(u'IARC pingback not of content-type "application/json"')
+            return Response({
+                'detail': "Endpoint only accepts 'application/json'."
+            }, status=http_status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+
+        app = self.get_object()
+        data = request.DATA[0]
```

Comment:
Since it's just a stub, maybe remove all that non v2 stuff instead ? It's confusing